tests/CacheManagerTest.php

```diff
@@ -2,17 +2,20 @@
 
 declare(strict_types=1);
 
+use Stancl\Tenancy\CacheManager;
+use Stancl\Tenancy\Tests\Etc\Tenant;
 use Illuminate\Support\Facades\Event;
-use Stancl\Tenancy\Bootstrappers\CacheTenancyBootstrapper;
 use Stancl\Tenancy\Events\TenancyInitialized;
 use Stancl\Tenancy\Listeners\BootstrapTenancy;
-use Stancl\Tenancy\Tests\Etc\Tenant;
+use Stancl\Tenancy\Bootstrappers\CacheTenancyBootstrapper;
 
 beforeEach(function () {
     config(['tenancy.bootstrappers' => [
         CacheTenancyBootstrapper::class,
     ]]);
 
+    CacheManager::$addTags = true;
```

Comment:
Can't this default to `true` and we'd just disable it in the tests specific to the cache prefix? Would require changing much fewer files I think. Remember that it's only a temporary thing we're adding before we send the Laravel PR, then we'll get rid of this completely.

Reply:
Yeah, that makes a lot more sense. Updated that